vkdgen.py

```diff
@@ -133,12 +133,12 @@ def __init__(self, errFile=sys.stderr, warnFile=sys.stderr, diagFile=sys.stderr)
 		self.functionVars = ""
 		self.opaqueStruct = set()
 		self.surfaceExtensions = {
-			"// VK_KHR_android_surface" : "VK_USE_PLATFORM_ANDROID_KHR",
-			"// VK_KHR_mir_surface"     : "VK_USE_PLATFORM_MIR_KHR",
-			"// VK_KHR_wayland_surface" : "VK_USE_PLATFORM_WAYLAND_KHR",
-			"// VK_KHR_win32_surface"   : "VK_USE_PLATFORM_WIN32_KHR",
-			"// VK_KHR_xlib_surface"    : "VK_USE_PLATFORM_XLIB_KHR",
-			"// VK_KHR_xcb_surface"	    : "VK_USE_PLATFORM_XCB_KHR",
+			"// VK_KHR_android_surface" : ["VK_USE_PLATFORM_ANDROID_KHR",	"public import android.native_window;\n"],
+			"// VK_KHR_mir_surface"     : ["VK_USE_PLATFORM_MIR_KHR",		"public import mir_toolkit.client_types;\n"],
+			"// VK_KHR_wayland_surface" : ["VK_USE_PLATFORM_WAYLAND_KHR",	"public import wayland_client;\n"],
+			"// VK_KHR_win32_surface"   : ["VK_USE_PLATFORM_WIN32_KHR",		"public import core.sys.windows.windows;\n"],
+			"// VK_KHR_xlib_surface"    : ["VK_USE_PLATFORM_XLIB_KHR",		"public import X11.Xlib;\n"],
+			"// VK_KHR_xcb_surface"	    : ["VK_USE_PLATFORM_XCB_KHR",		"public import xcb.xcb;\n"],
 		}

```

Comment:
Where are you getting these import paths from? The only one in the D standard library is the Windows one.